src/utils/misc.ts

```diff
@@ -1,6 +1,9 @@
 import { ethers } from 'ethers';
 import { keyBy, merge, values, orderBy, toNumber, isString } from 'lodash';
 
+import { createToken } from '@src/core/store/modules/tokens/tokens.selectors';
+import { Balance, Token, TokenView } from '@src/core/types';
```

Comment:
this should be `@types`

Reply:
@huggingbot yes, we try to use always the direct path alias where possible. You can consider refactoring this l around the PR